src/main/java/org/spongepowered/api/service/pagination/PaginationList.java

```diff
@@ -202,24 +246,30 @@ default void sendTo(MessageChannel channel) {
         PaginationList build();
 
         /**
-         * Send the constructed pagination list to the given receiver.
+         * Sends the constructed pagination list to the given receiver.
          *
-         * @param receiver The receiver to send to
+         * @param receiver The receiver to send the list to
          */
         default void sendTo(MessageReceiver receiver) {
             build().sendTo(receiver);
         }
 
         /**
-         * Send the constructed pagination list to the specified
-         * {@link MessageChannel}.
+         * Sends the constructed pagination list to the members of the
+         * specified {@link MessageChannel}.
          *
-         * @param channel channel to send to
+         * @param channel The channel to send the list to
          */
         default void sendTo(MessageChannel channel) {
             channel.getMembers().forEach(this::sendTo);
         }
 
+        /**
+         * Sends the constructed pagination list to an iterable of
+         * {@link MessageReceiver}s.
+         *
+         * @param receivers The message receivers to send the list to
+         */
         default void sendTo(Iterable<MessageReceiver> receivers) {
             receivers.forEach(this::sendTo);
```

Comment:
Oh you did not remove it from the main object it was missing in the first place. If we have the method on the builder it should be on the main as well.